packages/modular-scripts/src/cli.ts

```diff
@@ -118,6 +128,38 @@ function run() {
   }
 }
 
+interface WorkspaceMeta {
+  location: string;
+  workspaceDependencies: string[];
+  mismatchedWorkspaceDependencies: string[];
+}
+
+function listPackages(typeArg: string | void) {
+  const modularRoot = getModularRoot();
+
+  const { stdout } = execa.sync('yarnpkg', ['workspaces', 'info'], {
+    cwd: modularRoot,
+  });
+  const workspaceInfo: Record<string, WorkspaceMeta> = JSON.parse(
+    stdout,
+  ) as Record<string, WorkspaceMeta>;
+
+  const packages = Object.values(workspaceInfo)
+    .filter((workspaceMeta: WorkspaceMeta) => {
```

Comment:
Would this not work?

```suggestion
    .filter((workspaceMeta) => {
```